spec.bs

```diff
@@ -2800,6 +2803,143 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a `fetch()` call using the `{adAuctionHeaders: true}`
+option, which causes the value of the `Ad-Auction-Signals` response header on that request to
+be used to provide signals in auctions running on the page whose seller matches the origin of the
+`fetch()` destination.
+
+<div algorithm="fetch adAuctionHeaders patch">
+Modify [[FETCH]]'s [[FETCH#requests]] to add: 
+
+A [=request=] has an associated <dfn for=request>capture ad auction headers boolean</dfn>.
+Unless stated otherwise it is false.

```

Comment:
This should be worded like https://fetch.spec.whatwg.org/#request-keepalive-flag, and like the other associated boolean members.

Reply:
Done